CHANGELOG.md

```diff
@@ -11,8 +11,9 @@ CHANGELOG
 - Add support for multiple FSx File Systems.
 - Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
 - Add support for FSx Lustre Persistent_2 deployment type.
-- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
-- Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of the memory seen by the scheduler on compute nodes.
+- Add support for memory-based scheduling in Slurm.
```

Comment:
I think it's also important to add the following point from cookbook changelog:
Configure `RealMemory` on compute nodes by default as 95% of the EC2 memory.